docker/package/tezos_baking_wizard.py

```diff
@@ -0,0 +1,489 @@
+
+        proc_call("sudo -u tezos tezos-client bootstrapped")
+
+        print()
+        print("The Tezos node bootstrapped successfully.")
+
+    # Importing the baker key
+    def import_baker_key(self):
+        if self.config["key_import_mode"] == "secret-key":
+            self.check_baker_account(
+                "sudo -u tezos tezos-client import secret key baker "
+                + self.config["secret_key"]
+            )
+        elif self.config["key_import_mode"] == "json":
+            self.check_baker_account(
+                "sudo -u tezos tezos-client activate account baker with "
+                + self.config["json_filepath"]
+            )
+
+        else:
+            print()
+            input("Please open the Tezos Wallet app on your ledger, then hit Enter.")
+            self.check_baker_account(
+                "sudo -u tezos tezos-client import secret key baker "
+                + self.config["ledger_url"]
+            )
+
+            print()
+            input("Please open the Tezos Baking app on your ledger, then hit Enter.")
+            proc_call("sudo -u tezos tezos-client setup ledger to bake for baker")
+
+    def register_baker(self):
+        print()
+        input("Please open the Tezos Baking or Tezos Wallet app on your ledger, then hit Enter.")
+
+        proc_call("sudo -u tezos tezos-client register key baker as delegate")
+        print(
+            "You can check a blockchain explorer (e.g. https://tzkt.io/ or https://tzstats.com/) "
+            "to see the baker status and baking rights of your account."
+        )
+
+    def start_baking(self):
+        proc_call(
+            "sudo systemctl start tezos-baking-" + self.config["network"] + ".service"
+        )
+
+
+def run_setup(cfg):
+
+    if cfg is not None:
+        setup = Setup(cfg)
+
+        print("Trying to bootstrap tezos-node")
+        setup.bootstrap_node()
+
+        if cfg["mode"] == "node":
+            return
+
+        print("Importing the baker key")
+        setup.import_baker_key()
+
+        print("Registering the baker")
+        setup.register_baker()
+
+        print("Start baking instance")
+        setup.start_baking()
+
+        print(
+            "Congratulations! All required Tezos infrastructure services should now be started."
+            " You can show logs for all the services using the 'tezos' user by running: "
+            "journalctl -f _UID=$(id tezos -u)"
+        )
+
+        if cfg["mode"] == "baker":
+            print()
+            print(
+                "To stop the baking instance, run: sudo systemctl stop tezos-baking-"
+                + cfg["network"]
+                + ".service"
+            )
+
+
+if __name__ == "__main__":
+    readline.parse_and_bind("tab: complete")
+
+    networks = {
+        "mainnet": "Main Tezos network",
+        "florencenet": "Test network using version 9 of Tezos protocol (Florence)",
+    }
+
+    try:
+        cfg = run_wizard(networks)
```

Comment:
Not currently, so I agree with you there. I initially thought it might be possible to share the list of networks from the configuration, e.g. `packages.py`, but that's not true/easy from the way it will be used.